docs/release-process.md

```diff
@@ -245,8 +223,8 @@
     email.
 
 11. Commit and push all the tarballs and signature files to
-    `web/php-distributions.git`, then update the git submodule reference in
-    `web/php.git`:
+    `web-php-distributions.git`, then update the git submodule reference in
+    `web-php.git`:
```

Comment:
Several repo references above used just the name (e.g. web-php), but here you're using the .git suffix (web-php.git).  I'm fine either way, but we should pick one style and stick to it.

Reply:
Missed those, fixed ... gone with no .git suffix document wide